### DISCUSSION TITLE
Windows Powershell

### DISCUSSION THREAD

**Discussion by thistlillo** [starship/starship#2736] [Q&A] [answered]
I followed the installation instructions at:
https://starship.rs/
for installing starship in the Windows Powershell.

I have also installed (more than) a nerd font (and enabled it), but I ended up with the following problems:

- when I run the powershell (hit 'start', search 'powershell, then press enter), my prompt is not exactly what I was expecting:

![powershell-1](https://user-images.githubusercontent.com/6717655/118494231-3de32700-b722-11eb-8c46-1a03f18bea29.jpg)

- when I run the powershell inside of Windows Terminal, I get this error:
```
& : The term 'starship' is not recognized as the name of a cmdlet, function, script file, or operable program. Check
the spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Users\user\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1:1 char:21
+ Invoke-Expression (&starship init powershell)
+                     ~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (starship:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

What did I miss?

**Comment by davidkna** [comment, marked as answer]
The regular PowerShell window has some font rendering issues that are almost impossible to fix.

Windows Terminal should fix this. but it looks like the `starship.exe` is not in your `$Path`.
You can fix this either by using an absolute path to the starship binary `Invoke-Expression (&C:\path\to\starship_dir\starship.exe init powershell)` instead or by adding the directory with starship to your path before running the starship init. You can add it to your `$Path` via PowerShell in your `$PROFILE` with `$env:Path += ";C:\path\to\starship_dir"` or in the Windows settings.

If you have any further font rendering issues even in Windows Terminal please make sure you are using a [nerd font](https://www.nerdfonts.com/).

**Reply by thistlillo** [reply]
That solved the issue, thanks a lot.

![powershell2](https://user-images.githubusercontent.com/6717655/118499659-720d1680-b727-11eb-99a0-f4aa6fb43c18.jpg)

I think it would be worth mentioning the Powershell font rendering issue in the documentation: I have lost an hour trying mostly random stuff before coming here to ask :-D
